demos/2022.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: demos
+title: Community Demos 2022
+---
+
+# Community Demos (2022)
+------------------------------
+
+Every few weeks we host a Community Demo to showcase new & interesting developments from the Foreman
+community.
+We encourage participation from any member of the community (although you do need a
+Google account), so if you’ve been working on something cool, please do come show it off.
+If you'd like to watch the community demo, they're usually on a Thursday and we always list them in
+advance on our [Community Events Calendar](https://community.theforeman.org/c/events/l/calendar).
+
+Here is a list of every demo that has taken place this year.
+Our demos are split into two sections: a user-focused section and a developer-focused section.
```

Comment:
Should it also describe that user-focused demos are split into two again?

Reply:
Yep that's a good idea. I'll add it.